mods/sfinv/locale/sfinv.da.tr

```diff
@@ -0,0 +1,2 @@
+# textdomain: sfinv
+Crafting=
```

Comment:
Intentional or just forgotten? If intentional, maybe add a comment that this should not be translated?